kafka-ui-react-app/src/components/ACLPage/List/List.tsx

```diff
@@ -132,7 +132,7 @@ const ACList: React.FC = () => {
       <PageHeading text="Acsess Control List" />
       <Table
         columns={columns}
-        data={aclPayload ?? aclList ?? []}
+        data={aclList ?? []}
```

Comment:
please check the build and test errors / warnings `'aclPayload' is defined but never used`

Reply:
Oops. Sorry, remove unused import